### DISCUSSION TITLE
Changes to Airtable Attachments | Airtab... [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5861] [Discord Threads]
> 🙋‍♂️ **Dstala | NocoDB Team**

**Comment by o1lab** [comment]
> 🙋‍♂️ **saurav_mitra**  
#### Is there any way to expose the function that NocoDB uses to import an attachment from Airtable perhaps?   
#### Airtable webhooks don't give all the attachment data, just the attachment URL  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### will check with the team  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### You can check this   
#### https://all-apis.nocodb.com/#tag/Storage/operation/storage-upload-by-url  
#  
> 🙋‍♂️ **saurav_mitra**  
#### Yeah, that would be nice. the current problem with that is that the Airtable URL isn't directly to the attachment, but to a page that will display the attachment  
#  
> 🙋‍♂️ **saurav_mitra**  
#### But that'll be the step if I extract the data (like title and file data) out of the attachment  
#  
> 🙋‍♂️ **saurav_mitra**  
#### Any updates from the team regarding this?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1100606493864251473)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
